src/rviz/message_filter_display.h

```diff
@@ -58,6 +58,10 @@ Q_OBJECT
       topic_property_ = new RosTopicProperty( "Topic", "",
                                               "", "",
                                               this, SLOT( updateTopic() ));
+      unreliable_property_ = new BoolProperty( "Unreliable", false,
```

Comment:
I don't see a delete for `topic_property_` and that concerns me, or am I missing something?